tests/features/asset_distribution/add_solution.feature

```diff
@@ -0,0 +1,53 @@
+@api
+Feature: "Add asset distribution" visibility options.
+  In order to manage distributions
+  As a moderator
+  I need to be able to add "Asset distribution" rdf entities through UI.
+
+  Scenario: "Add distribution" button should only be shown to moderators.
+    Given the following collection:
+      | name | Asset Distribution Test                  |
+      | uri  | https://a.distribution/solution/add/test |
+      | logo | logo.png                                 |
+    And the following solution:
+      | name              | Asset random name                        |
+      | uri               | http://joinup.eu/ad/solution/api/bar     |
+      | description       | Some reusable random description         |
+      | elibrary creation | 1                                        |
+      | groups audience   | https://a.distribution/solution/add/test |
```

Comment:
This step does not match the behaviour of the functionality:

* The URI cannot be entered through the UI, so this should not be proposed as an option.
* "1" as an option for elibrary creation doesn't mean anything to an end user. This should be a human readable string such as "Only members can publish new content".
* "Groups audience" doesn't make sense from an end user perspective. This is a term from organic groups. This should be "collection" and it should take the human readable title of the collection, not the URI.